community-members.md

```diff
@@ -6,14 +6,14 @@ This is the current Governance Committee, per the [Governance Committee
 Charter](https://github.com/open-telemetry/community/blob/master/governance-charter.md),
 in alphabetical order:
 
-- [Alolita Sharma](https://github.com/alolita), AWS, until October 2022
+- [Alolita Sharma](https://github.com/alolita), AWS, until October 2024
 - [Ben Sigelman](https://github.com/bhs), Lightstep, until October 2023
 - [Bogdan Drutu](https://github.com/BogdanDrutu), Splunk, until October 2023
-- [Daniel Dyla](https://github.com/dyladan), Dynatrace, until October 2022
+- [Daniel Dyla](https://github.com/dyladan), Dynatrace, until October 2024
 - [Juraci Paixão Kröhling](https://github.com/jpkrohling), Grafana Labs, until October 2023
-- [Liz Fong-Jones](https://github.com/lizthegrey), Honeycomb, until October 2022
```

Comment:
add to Emeritus section https://github.com/open-telemetry/community/blob/24b30b978af4a71066655c62dfc50a2e60101d06/community-members.md?plain=1#L163

Reply:
Great catch. Fixed.